apps/aecore/src/aecore_app.erl

```diff
@@ -14,14 +14,21 @@
 
 start(_StartType, _StartArgs) ->
     ok = lager:info("Starting aecore node"),
+    ok = application:ensure_started(mnesia),
     aecore_sup:start_link().
 
+start_phase(load_database, _StartType, _PhaseArgs) ->
+    lager:debug("start_phase(load_database, _, _)", []),
+    aec_db:load_database();
 start_phase(create_metrics_probes, _StartType, _PhaseArgs) ->
+    lager:debug("start_phase(create_metrics_probes, _, _)", []),
     aec_metrics:create_metrics_probes();
 start_phase(start_reporters, _StartType, _PhaseArgs) ->
+    lager:debug("start_phase(start_reporters, _, _)", []),
     aec_metrics_rpt_dest:check_config(),
     aec_metrics:start_reporters().
 
+
 stop(_State) ->
```

Comment:
Shouldn't we stop mnesia as well?

Reply:
There would be a point to that. In general, I would object that the only reason it's started by aecore is to respect a start dependency; otherwise, mnesia should be regarded as started in its own right. In practice, it shouldn't really matter (except that some unit tests have to explicitly stop mnesia.)